docker/README.md

```diff
@@ -12,11 +12,7 @@ By default the following config.json overrides are applied:
 
 | Setting | Value |
 | ------- | ----- |
-| GossipFanout | 1 |
 | EndpointAddress | 0.0.0.0:8080 |
-| IncomingConnectionsLimit | 0 |
-| Archival | false |
-| IsIndexerActive | false |
 | EnableDeveloperAPI | true |
 
 ### Environment Variables
```

Comment:
The explicit variable checks in the table below should be updated from `If set,` to something like `When set to 1,`